client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-conflicts.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+
+class DomainConnectAuthorizeConflicts extends Component {
+	placeholder = () => {
+		return (
+			<div>
+				<div className="domain-connect__is-placeholder">
+					<span></span>
+					<span></span>
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { isPlaceholder, conflictingRecords } = this.props;
```

Comment:
Huh, didn't realize that so many components already used that convention - very good point, consistency is important :) Seems fine to keep it `isPlaceholder` then.